website/docs/reference/feature-toggle-variants.md

```diff
@@ -1,23 +1,23 @@
 ---
 title: Feature Toggle Variants
 ---
-<div class="alert alert--info" role="alert">
-  <em>Feature toggle variants</em> were introduced in <em>Unleash v3.2.0</em>.
-</div>
-<br/>
+:::info Availability
+  **Feature toggle variants** were introduced in Unleash 3.2.
+  **Per-environment variants** were introduced in Unleash 4.21.
+:::
```

Comment:
```suggestion
:::info Availability

**Feature toggle variants** were first introduced in Unleash 3.2.

The option to have **different variant configurations in each environment** was introduced in Unleash 4.21.

:::
```

Reply:
I like it better, but I'm not sure I'd call it an option, because it's not optional. You have to deal with different variants configurations in each environment

Reply:
Ah, that's a very good point. I mean, technically, you can have the same variants for all environments still: you just need to keep them in sync. But I agree that it's confusing and potentially misleading 😅 

How about this?

```suggestion
:::info Availability

**Feature toggle variants** were first introduced in Unleash 3.2.
In Unleash 4.21, variants were updated to be **environment-dependent**, meaning the same feature could have different variant configurations in different environments.

:::
```